selfdrive/car/gm/carstate.py

```diff
@@ -26,6 +26,7 @@ def update(self, pt_cp, cam_cp, loopback_cp):
     self.prev_cruise_buttons = self.cruise_buttons
     self.cruise_buttons = pt_cp.vl["ASCMSteeringButton"]["ACCButtons"]
     self.buttons_counter = pt_cp.vl["ASCMSteeringButton"]["RollingCounter"]
+    self.pscm_status = pt_cp.vl["PSCMStatus"]
```

Comment:
why did you remove the copy? this overwrites the dict now

Reply:
It's fine if we do, but I can add it back so it's explicit